app/controllers/api/authorizations_controller.rb

```diff
@@ -0,0 +1,20 @@
+# encoding: utf-8
+
+class Api::AuthorizationsController < Api::BaseController
+  def resource
+    Authorization.new(User.find(prx_auth_token.user_id))
```

Comment:
I might DRY this out a bit and put `User.find(prx_auth_token.user_id)` into a find or get user method.